nexus/tests/integration_tests/authz.rs

```diff
@@ -0,0 +1,439 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Tests for authz policy not covered in the set of unauthorized tests
+
+use nexus_test_utils::http_testing::{AuthnMode, NexusRequest, RequestBuilder};
+use nexus_test_utils::ControlPlaneTestContext;
+use nexus_test_utils_macros::nexus_test;
+
+use omicron_common::api::external::IdentityMetadataCreateParams;
+use omicron_nexus::external_api::params;
+use omicron_nexus::external_api::shared;
+use omicron_nexus::external_api::views;
+use omicron_nexus::TestInterfaces;
+
+use dropshot::ResultsPage;
+use nexus_test_utils::resource_helpers::create_silo;
+
+use httptest::{matchers::*, responders::*, Expectation, ServerBuilder};
+
+use uuid::Uuid;
+
+// Test that an authenticated, unprivileged user has full CRUD access to their SSH keys
```

Comment:
Is this not tested already by the sshkey tests?  (Or, could they be, if those tests used `UnprivilegedUser`?)  (cc @plotnick)

Reply:
I think so, yes. It looks like creation, listing, and deletion are all covered by the tests below, but as you say not by an unprivileged user.

Reply:
Addressed in #1399.